src/rgw/rgw_sal_motr.cc

```diff
@@ -3314,7 +3351,6 @@ void MotrStore::index_name_to_motr_fid(string iname, struct m0_uint128 *id)
   hash.SetFlags(EVP_MD_CTX_FLAG_NON_FIPS_ALLOW);
   hash.Update((const unsigned char *)iname.c_str(), iname.length());
   hash.Final(md5);
-
```

Comment:
Revert please.

Reply:
Thanks for taking care of this in https://github.com/Seagate/cortx-rgw/pull/85